lib/open_telemetry.ex

```diff
@@ -1,4 +1,11 @@
 defmodule OpenTelemetry do
-  defdelegate generate_trace_id(), to: :opentelemetry
-  defdelegate generate_span_id(), to: :opentelemetry
+  # Helpers to build OpenTelemetry structured types
```

Comment:
Do these automatically pick up the specs of the functions they delegate to? Like for documentation? If not, and maybe even if, we may want to add specs and give the types names, as that is great for autocomplete and docs.

Reply:
Don't know. That would be nice if they do.

Reply:
If you generate the hex docs you should be able to look at that module and see

Reply:
Checked, and it does a `See :opentelemetry.event/2.`

But it can't do docs for erlang modules, right? Meaning we need to doc them separately and hopefully be able to get rid of the `see` part of the hex docs.

Reply:
> But it can't do docs for erlang modules, right?

Soon. :) You can explicitly doc them for now and have a complete link to where the Erlang docs are hosted. Basically, doc them like this:

```
See [`:opentelemetry.event/2`](https://idontuseexdoc/yet.html).
```

Reply:
Yea, we are going to publish docs for the Erlang part anyway so can do that. Need to spend the time soon to work out publishing them in parallel on the same app to hex.